app/javascript/pages/country/widget/widgets/widget-fao-forest/widget-fao-forest-styles.scss

```diff
@@ -0,0 +1,28 @@
+@import '~styles/settings.scss';
+
+.c-widget-fao-forest {
+  .sentence {
```

Comment:
Wonder if we can make the sentence a really simple component <Sentence> so we dont have to style is every time?